tests/python/rest_api/_test_base.py

```diff
@@ -510,6 +511,8 @@ def _uploaded_video_task_fxt_base(
         start_frame: int | None = None,
         stop_frame: int | None = None,
         step: int | None = None,
+        video_file: IO[bytes] | None = None,
```

Comment:
Perhaps instead of this it would be better to have a parameter like `video_kwargs: dict | None`? `fxt_uploaded_video_task_without_manifest` would then look like:

```
        yield from self._uploaded_video_task_fxt_base(
            request=request, video_kwargs={"invalid_keyframes": True}, chapters=[]
        )
```

And the logic in this function would become simpler.